sqlglot/dialects/postgres.py

```diff
@@ -269,6 +269,7 @@ class Parser(parser.Parser):
             "NOW": exp.CurrentTimestamp.from_arg_list,
             "TO_CHAR": format_time_lambda(exp.TimeToStr, "postgres"),
             "TO_TIMESTAMP": _to_timestamp,
+            "UNNEST": lambda args: exp.Unnest(expressions=[seq_get(args, 0)]),
```

Comment:
I don't know where you draw the line on when to extract this into a function, but I reckon this is fine to inline?